frontend/src/utils.js

```diff
@@ -0,0 +1,68 @@
+import { v4 as uuidv4 } from "uuid";
+function chillHop() {
+  return [
+    {
+      name: "Beaver Creek",
+      cover:
+        "https://chillhop.com/wp-content/uploads/2020/09/0255e8b8c74c90d4a27c594b3452b2daafae608d-1024x1024.jpg",
+      artist: "Aso, Middle School, Aviino",
+      audio: "Beaver Creek.mp3",
+      color: ["#205950", "#2ab3bf"],
+      id: uuidv4(),
+      active: true,
+    },
+    {
```

Comment:
You could just make this a const object like 

```js
export const songData = [
{
// Your song data here
}
]
```

Whatever consumer is using `util.js` you could do

```js
import { songData } from "./util.js"
```

Reply:
good idea.